Allow cluster to recover during partial storage failure

drdivano opened the pull request:
podManagementPolicy: Parallel prevents the following situation:

1. We run Stolon as a StatefulSet with 2 or more replicas
2. The cluster goes down together with storage system
3. The cluster comes up. The storage system comes up partially, with Persistent Volume 0 still unavailable.
4. Kubernetes fails to re-schedule and start the StatefulSet since storage for replica 0 is still down.

The Parallel policy tells Kubernetes to start other replicas without waiting for replicas with lower ordinal numbers.

sgotti commented:
@drdivano Thanks for the PR!. Since it's related to just the k8s example can you rename it to `k8s example: use podManagementPolicy: Parallel`

And then in the description describe why it's useful.

drdivano commented:
> @drdivano Thanks for the PR!. Since it's related to just the k8s example can you rename it to k8s example: use podManagementPolicy: Parallel
> 
> And then in the description describe why it's useful.

@sgotti I renamed the PR and added a description as you asked 13 days ago. Did I misunderstand you?

sgotti commented:
> @sgotti I renamed the PR and added a description as you asked 13 days ago. Did I misunderstand you?

Sorry, I just lost your push. Can you please rebase in a single commit instead of merging master branch (for a cleaner git history)? Thanks.
